apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/define/StatementEnhanceInfos.java

```diff
@@ -38,6 +38,9 @@ public StatementEnhanceInfos(ConnectionInfo connectionInfo, String sql, String s
         this.statementName = statementName;
     }
 
+    public StatementEnhanceInfos() {
```

Comment:
Why do you build an empty one? You can't get the necessary information?

Reply:
Usually, if these information is available, it is better to build in the constructor.
Otherwise, one day, we may face uncertain NPE.

Reply:
i can get the information, but it is useless, i just use param set part,  ok, i will use the constructor and set  information to it